packages/fxa-event-broker/package.json

```diff
@@ -3,13 +3,18 @@
   "version": "1.185.1",
   "description": "Firefox Accounts Event Broker",
   "scripts": {
-    "build": "tsc",
-    "lint": "eslint *",
     "audit": "npm audit --json | audit-filter --nsp-config=.nsprc --audit=-",
-    "watch": "tsc -w",
-    "test": "mocha -r ts-node/register src/test/**/*.spec.ts src/test/**/**/*.spec.ts",
-    "start-prod": "npm run build && node ./dist/bin/worker.js",
+    "prebuild": "rimraf dist",
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "../../_scripts/check-url.sh localhost:9000/__heartbeat__ && pm2 start pm2.config.js",
+    "start:prod": "node dist/main",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "jest",
```

Comment:
We're moving to Jest for testing here? [I thought we were only using Jest for front-end React stuff?](https://github.com/mozilla/fxa/blob/main/docs/adr/0009-testing-stacks.md#identify-a-consistent-stack-of-testing-tools-per-aspect-of-the-project) 

Assuming that's changing along with NestJS, we might want to draft an ADR to obsolete 0009 and explain why

Reply:
Yea, this came up in the NestJS ADR, I agree it should be formalized as a new ADR. Danny mentioned it as, "We should stick with NestJS defaults" on the architecture ADR, which is the brunt of it. I think that also implies a formalization of using Express, as that's the default web server used by NestJS as well.

Reply:
Could even edit / expand the existing NestJS ADR to explicitly say "stick with NestJS defaults" means Jest & Express.

Normally, I'd say a new ADR is better. But, if these things were already understood & implicit in the discussion of 0020, it's worth just elaborating in the existing ADR. Then maybe link from 0009 as it being obsoleted by 0020.

I thought I remembered us drafting an ADR around Hapi vs Express, but doesn't seem like we did that. Otherwise, I'd say that could use a link too

Reply:
I'll update the ADR with notes to use the NestJS default of Express+Jest.